common/script/index.js

```diff
@@ -33,10 +33,6 @@ api.capByLevel = statHelpers.capByLevel;
 api.tnl = statHelpers.toNextLevel;
 api.diminishingReturns = statHelpers.diminishingReturns;
 
-// TODO under api.libs?
-import splitWhitespace from './libs/splitWhitespace';
-const $w = api.$w = splitWhitespace;
```

Comment:
Are you sure? I can add it back with a linter exception. If it's used on the frontend in another file, wouldn't linter catch it with "$w is undefined"?